airflow/utils/state.py

```diff
@@ -137,6 +137,20 @@ def color_fg(cls, state):
             return "white"
         return "black"
 
+    waiting: frozenset[TaskInstanceState] = frozenset(
+        [TaskInstanceState.DEFERRED, TaskInstanceState.UP_FOR_RESCHEDULE]
+    )
+    """
+    A list of states indicating that a task is sensing or deferred
+    """
+
+    pending: frozenset[TaskInstanceState] = frozenset(
+        [TaskInstanceState.RUNNING, TaskInstanceState.DEFERRED, TaskInstanceState.UP_FOR_RESCHEDULE]
+    )
+    """
+    Running tasks + sensing tasks (task reschedules makes up_for_reschedule act a little differently)
+    """
+
```

Comment:
I think it's best not to add these.  I've seen recently how these types of labels can mean different things in different contexts.  E.g. we have `DEFERRED` as part of the `running` list, and that results in the wrong try number when logging from trigger.  I think when in doubt just reference the states explicitly.

Reply:
updated